worker/caasfirewallersidecar/worker.go

```diff
@@ -118,6 +119,19 @@ func (p *firewaller) loop() error {
 				return errors.New("watcher closed channel")
 			}
 			for _, appName := range apps {
+				// If charm is (now) a v1 charm, skip processing.
```

Comment:
"(now)" is not really relevant side a v2 charm can never transition to a v1 charm

Reply:
Good call, removed "(now)"